services/storage/src/simcore_service_storage/api/v0/openapi.yaml

```diff
@@ -1191,6 +1191,11 @@ components:
           type: array
           items:
             type: integer
+        folder:
```

Comment:
Adding a folder property of the project is an unnecessary dependency. Take the analogy with a file system where the files are equivalent to projects and the directories  to your folders. The file knows nothing about which folder is contained in. Instead there is a table that creates that hierarchy "from the top". The dashboard will be your root directory and there you have files and folders. Accessing to meta-info on the folder can be aggregated at the API level by picking necessary info from the projects table. In addition, populating more the projects table will interfere with other parts of the code (those are the CI errors you are getting) which will require doing extra changes that even might affect performance of those other parts.

Think of the folder resource as an abstraction on top of projects that is defined *per user* (actually I would even associate that a groupID to be even more generic). I think the starting point of having a separated table for folders that you already have is the right way to go. Try to represent the hierarchy there and associate some entries with project uuids. I googled and found some basic concept https://patshaughnessy.net/2017/12/11/trying-to-represent-a-tree-structure-using-postgres

then in the handlers resolve the tree and aggregate the info you want from the projects table